tests/test_extending_bakery.py

```diff
@@ -1,14 +1,17 @@
 import pytest
+
 from model_bakery import baker
 from model_bakery.exceptions import CustomBakerNotFound, InvalidCustomBaker
-from model_bakery.random_gen import gen_from_list
+from model_bakery.random_gen import action_generator, gen_from_list
 from tests.generic.models import Person
 
 
+@action_generator
```

Comment:
This is an example of what I was trying to say in my previous comment. The existing custom generators API will break =/